### ISSUE TITLE
Adding New System Requirements

### ISSUE THREAD

**Issue by ertbulbul** [bounswe/bounswe2020group7#31]
We need to edit system requirements by making new additions after the newly created classification.

* **Deadline: 29.02.2020 23.59**

**Comment by oykuyilmaz1** [comment]
- [ ] Alperen Divriklioglu

- [ ] Mehmet Temizel

- [ ] Umut Ozdemir

- [ ] Ertugrul Bulbul

- [ ] Oyku Yilmaz

**Comment by oykuyilmaz1** [comment]
I added some requirements to events, home page, search, and notifications

**Comment by HUmutOzdemir** [comment]
* I added the following requirements:
1.2.2.6, 1.2.3.7, 1.2.3.8, 1.2.4.5, 1.2.4.6, 1.2.5.7, 1.2.5.8, , all requirements of 1.2.6,1.2.7.7
* Also I changed some of the existing requirements:
1.2.1.2 => In a tab or page part is an answer to "How?" question
1.2.2.3 => A calendar like bar part is an answer to "How?" question
1.2.2.4 => A bar part is an answer to "How?" question
1.2.2.5 => Have a bar part is an answer to "How?"
* Also, I add new things to glossary part:
Upcoming Event,  Filter(Search), Search for Collaborators State, Ongoing State,  Published State, Semantic Search,  Advanced Search, Application to the Project, Team of a Workspace/Project, Visible, Invisible, Project Related Materials

* Please check and give feedback about my changes.

**Comment by ertbulbul** [comment]
I checked, everything looks right. Good job.

**Comment by MEHMETTEMIZEL** [comment]
I checked also, it looks well and errorless.
By the way I reordered the glossary part as an alphabetic sequence.

**Comment by darktheorys** [comment]
I also reviewed, it is very good. Also, **i think** that "**upcoming events**" section is very **narrow**. There can be requirements about **conference, journal details and deadlines**. If one is enough for you, i think we can **delete this title and redistribute it**.
 It is up to you guys.

**Comment by alperendivriklioglu** [comment]
Having read the project description and customer meeting notes several times, I think the requirements list is prepared well. So, I did not feel a need to modify it a lot. 

I only added **a new requirement** for the projects: a project should be linkable to an upcoming event in the system (conference, journal etc.); and when a project is linked to an event, the deadline of that project will automatically be set as the deadline for submission of that particular event. Therefore, the academicians will not need to set the deadlines of their projects manually and they will be able to see which of their "projects" are related to which event (conference etc.)
